rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/util/RMProxyUserInterface.java

```diff
@@ -492,4 +503,19 @@ public void setNodeTokens(String nodeUrl, List<String> tokens) throws RMExceptio
     public List<String> getNodeTokens(String nodeUrl) throws RMException {
         return target.getNodeTokens(nodeUrl);
     }
+
+    @Override
+    public Set<String> getNodeTags(String nodeUrl) throws RMException {
+        return target.getNodeTags(nodeUrl);
+    }
+
+    @Override
+    public Set<String> getNodesByTags(String tag) {
+        return target.getNodesByTags(tag);
+    }
+
+    @Override
+    public Set<String> getNodesByTags(List<String> tags, boolean all) {
```

Comment:
why not using Set<String> tags, even if the rest request has a list parameter, you can convert a list to a set, and it will ensure unicity of the tag in the backend